src/mass/index.ts

```diff
@@ -18,6 +18,8 @@ export function withValueType<NumberType>(arithmetic: Arithmetic<NumberType>) {
     static kilograms = makeUnit('kg', dimension.Mass);
 
     static grams = WithValueType.kilograms.times(1e-3).withSymbol('g');
+
+    static pounds = WithValueType.kilograms(4.5359237e-1).withSymbol('lb');
```

Comment:
Should be `WithValueType.kilograms.times(4.5359237e-1).withSymbol('lb');`

Reply:
I'm not sure what I did here, but I'm somewhat sorry for all the noise. Let's hope it works this time.

Reply:
No worries at all. Thanks a lot for the contribution!